src/core/algorithms/statistics/data_stats.cpp

```diff
@@ -606,6 +623,295 @@ Statistic DataStats::GetAvgNumberOfChars(size_t index) const {
     return Statistic(res, &double_type, false);
 }
 
+Statistic DataStats::GetMinChars(size_t index) const {
+    if (all_stats_[index].min_chars.HasValue()) return all_stats_[index].min_chars;
```

Comment:
Try get this statistic from `GetMin` if it was calculated. The same for `GetMax` and `GetMaxChars`.

Reply:
It seems to work just fine in C++, but in python get_min() and get_max() seem to order strings alphabetically, so the minimal string would be the one starting with "aA", for example, not the shortest one. I think I'd stick to taking the minimum of string sizes

Reply:
Another solution might be writing additional variants of metrics without checking GetMin and GetMax, and make bindings with them rather than "original" methods, but I feel like it'll make code more convoluted than it should be, + if this problem arose with python it might arise somewhere else